--- ChangeLog	2024-12-19 07:53:08.353663909 -0500
+++ ChangeLog-spell-corrected	2024-12-19 07:52:45.304655652 -0500
@@ -110,7 +110,7 @@
 	* .pre-commit-config.yaml, xdis/instruction.py: tolerates compatible
 	Instruction nametuples The control-flow project uses a compatible Instruction nametuple,
 	ExtendedInstruction.  To make this possible, we should not call Instruction, but
-	self.__class__ to accomodate such extension.
+	self.__class__ to accommodate such extension.
 
 2024-11-09  rocky <rocky@dusytfeet.com>
 
@@ -225,7 +225,7 @@
 	admin-tools/pyenv-3.1-3.2-versions,
 	admin-tools/pyenv-3.3-3.5-versions,
 	admin-tools/pyenv-3.6-3.10-versions,
-	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.1.sh: 
+	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.1.sh:
 	Administrivia
 
 2024-10-08  elliot <elliot@utdallas.edu>
@@ -460,12 +460,12 @@
 	* admin-tools/make-dist-3.0-3.2.sh,
 	admin-tools/make-dist-3.3-3.5.sh,
 	admin-tools/make-dist-3.6-3.10.sh, admin-tools/setup-master.sh,
-	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh: 
+	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia
 
 2024-07-22  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/setup-master.sh, admin-tools/setup-python-3.6.sh: 
+	* admin-tools/setup-master.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia
 
 2024-07-21  rocky <rocky@dusytfeet.com>
@@ -493,7 +493,7 @@
 
 2024-07-21  rocky <rocky@dusytfeet.com>
 
-	* xdis/magics.py: Note 3.12.4's existance
+	* xdis/magics.py: Note 3.12.4's existence
 
 2024-07-21  rocky <rocky@dusytfeet.com>
 
@@ -558,7 +558,7 @@
 2024-07-11  rocky <rocky@dusytfeet.com>
 
 	* xdis/opcodes/format/extended.py, xdis/opcodes/opcode_2x.py,
-	xdis/opcodes/opcode_35.py: Need to split BUILD_MAP ...  From pre 3.5 version with older 2.x vesion
+	xdis/opcodes/opcode_35.py: Need to split BUILD_MAP ...  From pre 3.5 version with older 2.x version
 
 2024-07-11  rocky <rocky@dusytfeet.com>
 
@@ -720,7 +720,7 @@
 
 2024-06-27  rocky <rocky@dusytfeet.com>
 
-	* xdis/codetype/code310.py: explit yield instead of yield via return
+	* xdis/codetype/code310.py: explicit yield instead of yield via return
 
 2024-06-27  rocky <rocky@dusytfeet.com>
 
@@ -898,7 +898,7 @@
 	xdis/opcodes/opcode_36pypy.py, xdis/opcodes/opcode_37.py,
 	xdis/opcodes/opcode_37pypy.py, xdis/opcodes/opcode_38pypy.py,
 	xdis/opcodes/opcode_39pypy.py, xdis/opcodes/opcode_3x.py: Add a
-	"call_op", improve formatting of CALL_METHOD CALL_METHOD is simliar to CALL_FUNCTION tag opcodes that are
+	"call_op", improve formatting of CALL_METHOD CALL_METHOD is similar to CALL_FUNCTION tag opcodes that are
 	call_ops that perform some sort of call
 
 2024-05-27  R. Bernstein <rocky@users.noreply.github.com>
@@ -982,7 +982,7 @@
 
 2024-03-31  rocky <rocky@dusytfeet.com>
 
-	* xdis/opcodes/opcode_36.py: Simplfy "Neither x, nor y, nor" to "No
+	* xdis/opcodes/opcode_36.py: Simplify "Neither x, nor y, nor" to "No
 	arguments'
 
 2024-03-29  rocky <rocky@dusytfeet.com>
@@ -1009,12 +1009,12 @@
 2024-03-16  rocky <rocky@dusytfeet.com>
 
 	* pyproject.toml, xdis/version.py: Remove unused dependencies; bump
-	verison
+	version
 
 2024-03-15  rocky <rocky@dusytfeet.com>
 
 	* admin-tools/make-dist-2.4-2.7.sh,
-	admin-tools/make-dist-3.0-3.2.sh, admin-tools/make-dist-newest.sh: 
+	admin-tools/make-dist-3.0-3.2.sh, admin-tools/make-dist-newest.sh:
 	Administrivia
 
 2024-03-15  rocky <rocky@dusytfeet.com>
@@ -1225,7 +1225,7 @@
 2024-02-12  rocky <rocky@dusytfeet.com>
 
 	* admin-tools/merge-for-2.4.sh, admin-tools/merge-for-3.0.sh,
-	admin-tools/merge-for-3.3.sh, admin-tools/merge-for-3.6.sh: 
+	admin-tools/merge-for-3.3.sh, admin-tools/merge-for-3.6.sh:
 	Administrivia: merge should leave us back in the directory we started
 
 2024-02-12  rocky <rocky@dusytfeet.com>
@@ -1687,7 +1687,7 @@
 
 2023-08-31  rocky <rocky@dusytfeet.com>
 
-	* xdis/opcodes/format.py, xdis/opcodes/opcode_36.py: imporve CALL
+	* xdis/opcodes/format.py, xdis/opcodes/opcode_36.py: improve CALL
 	argument detection...  merge CALL_FUNCTION36 and CALL_FUNCTION
 
 2023-08-31  rocky <rocky@dusytfeet.com>
@@ -1706,7 +1706,7 @@
 
 2023-08-30  R. Bernstein <rocky@users.noreply.github.com>
 
-	* : Merge pull request #116 from rocky/format-with-TOS Revise extended format incorprating TOS
+	* : Merge pull request #116 from rocky/format-with-TOS Revise extended format incorporating TOS
 
 2023-08-28  rocky <rocky@dusytfeet.com>
 
@@ -1846,7 +1846,7 @@
 2023-08-20  rocky <rocky@dusytfeet.com>
 
 	* xdis/magics.py, xdis/op_imports.py,
-	xdis/opcodes/opcode_310pypy.py, xdis/opcodes/opcode_39pypy.py: 
+	xdis/opcodes/opcode_310pypy.py, xdis/opcodes/opcode_39pypy.py:
 	Minimal PyPy 3.10 support
 
 2023-08-20  rocky <rocky@dusytfeet.com>
@@ -1862,7 +1862,7 @@
 2023-08-20  rocky <rocky@dusytfeet.com>
 
 	* xdis/opcodes/base.py, xdis/opcodes/opcode_311.py: 3.11 LOAD_ATTR
-	extended formatting...  ignore CACHE instructions.  And remove BINARY_xxx rutines. 3.11 uses
+	extended formatting...  ignore CACHE instructions.  And remove BINARY_xxx routines. 3.11 uses
 	BINARY_OP
 
 2023-08-20  rocky <rocky@dusytfeet.com>
@@ -1969,7 +1969,7 @@
 
 2023-06-16  rocky <rocky@dusytfeet.com>
 
-	* xdis/op_imports.py: stdlib use major minor when opcode lookup..  and we cant find major/minor release
+	* xdis/op_imports.py: stdlib use major minor when opcode lookup..  and we can't find major/minor release
 
 2023-06-14  rocky <rocky@dusytfeet.com>
 
@@ -2003,7 +2003,7 @@
 
 2023-06-01  rocky <rocky@dusytfeet.com>
 
-	* xdis/magics.py, xdis/opcodes/base.py, xdis/opcodes/opcode_311.py: 
+	* xdis/magics.py, xdis/opcodes/base.py, xdis/opcodes/opcode_311.py:
 	3.11 first working cut
 
 2023-06-01  rocky <rocky@dusytfeet.com>
@@ -2188,7 +2188,7 @@
 2022-12-07  rocky <rocky@dusytfeet.com>
 
 	* .github/workflows/osx.yml, .github/workflows/ubuntu.yml,
-	.github/workflows/windows.yml: CI workflows woes - No Pyton 3.6
+	.github/workflows/windows.yml: CI workflows woes - No Python 3.6
 
 2022-12-07  rocky <rb@dustyfeet.com>
 
@@ -2285,7 +2285,7 @@
 
 2022-06-16  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/pyenv-newest-versions, xdis/instruction.py: 
+	* admin-tools/pyenv-newest-versions, xdis/instruction.py:
 	Administrivia
 
 2022-06-16  rocky <rocky@dusytfeet.com>
@@ -2313,7 +2313,7 @@
 2022-05-19  rocky <rocky@dusytfeet.com>
 
 	* NEWS.md, admin-tools/pyenv-newest-versions, xdis/magics.py,
-	xdis/op_imports.py, xdis/opcodes/opcode_39pypy.py, xdis/version.py: 
+	xdis/op_imports.py, xdis/opcodes/opcode_39pypy.py, xdis/version.py:
 	Add 3.9.13 PyPy 3.9 & get ready for release 6.0.4
 
 2022-05-14  rocky <rocky@dusytfeet.com>
@@ -2365,7 +2365,7 @@
 2022-03-16  rocky <rocky@dusytfeet.com>
 
 	* .github/workflows/osx.yml, .github/workflows/ubuntu.yml,
-	.github/workflows/windows.yml, admin-tools/pyenv-newest-versions: 
+	.github/workflows/windows.yml, admin-tools/pyenv-newest-versions:
 	Use v3 checkout on workflows CI
 
 2022-03-11  rocky <rocky@dusytfeet.com>
@@ -2374,7 +2374,7 @@
 
 2022-03-10  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/check-3.3-3.5-versions.sh, xdis/opcodes/opcode_37.py: 
+	* admin-tools/check-3.3-3.5-versions.sh, xdis/opcodes/opcode_37.py:
 	Misc small tweaks
 
 2022-03-09  rocky <rocky@dusytfeet.com>
@@ -2421,7 +2421,7 @@
 
 2021-11-27  rocky <rocky@dusytfeet.com>
 
-	* appveyor.yml, appveyor/install.ps1, appveyor/run_with_env.cmd: 
+	* appveyor.yml, appveyor/install.ps1, appveyor/run_with_env.cmd:
 	Remove appveyor
 
 2021-11-27  R. Bernstein <rocky@users.noreply.github.com>
@@ -2443,7 +2443,7 @@
 
 2021-11-24  rocky <rocky@dusytfeet.com>
 
-	* xdis/opcodes/opcode_37pypy.py, xdis/opcodes/opcode_38pypy.py: 
+	* xdis/opcodes/opcode_37pypy.py, xdis/opcodes/opcode_38pypy.py:
 	Include PyPy 3.8 CALL_METHOD{,_KW} formatting
 
 2021-11-24  rocky <rocky@dusytfeet.com>
@@ -2676,7 +2676,7 @@
 
 2021-10-12  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/pyenv-newest-versions, pytest/test_instructions.py: 
+	* admin-tools/pyenv-newest-versions, pytest/test_instructions.py:
 	Last 3.10.0 test adjusted
 
 2021-10-11  rocky <rocky@dusytfeet.com>
@@ -2781,7 +2781,7 @@
 
 	* xdis/bin/pydisasm.py, xdis/load.py, xdis/magics.py,
 	xdis/op_imports.py, xdis/opcodes/base.py,
-	xdis/opcodes/opcode_310.py: Start adjusting for Pyton 3.10 Major mess ensuse because 3.10 can't be used as a floating-point
+	xdis/opcodes/opcode_310.py: Start adjusting for Python 3.10 Major mess ensuse because 3.10 can't be used as a floating-point
 	number.
 
 2021-09-30  rocky <rocky@dusytfeet.com>
@@ -2799,7 +2799,7 @@
 
 2021-09-21  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/make-dist-3.3-3.5.sh, admin-tools/make-dist-newer.sh: 
+	* admin-tools/make-dist-3.3-3.5.sh, admin-tools/make-dist-newer.sh:
 	Fix wheel packaging
 
 2021-09-21  rocky <rocky@dusytfeet.com>
@@ -2853,7 +2853,7 @@
 	* admin-tools/{check-older-versions.sh =>
 	check-2.4-2.7-versions.sh}, admin-tools/{pyenv-oldest-versions =>
 	pyenv-2.1-2.3-versions}, admin-tools/{pyenv-older-versions =>
-	pyenv-2.4-2.7-versions}, admin-tools/setup-python-2.4.sh: 
+	pyenv-2.4-2.7-versions}, admin-tools/setup-python-2.4.sh:
 	Adminsitrivia More version setup renaming
 
 2021-09-11  rocky <rocky@dusytfeet.com>
@@ -2936,7 +2936,7 @@
 
 2021-09-08  rocky <rocky@dusytfeet.com>
 
-	* xdis/unmarshal.py: Remove duplcate method in unmarshal Note string_as_bytes weirdness...
+	* xdis/unmarshal.py: Remove duplicate method in unmarshal Note string_as_bytes weirdness...
 
 2021-09-01  rocky <rocky@dusytfeet.com>
 
@@ -3058,7 +3058,7 @@
 
 2021-02-27  rocky <rocky@dusytfeet.com>
 
-	* : Python 3.1.5 compatiability
+	* : Python 3.1.5 compatibility
 
 2021-02-27  rocky <rocky@dusytfeet.com>
 
@@ -3067,11 +3067,11 @@
 
 2021-02-27  rocky <rocky@dusytfeet.com>
 
-	* xdis/op_imports.py: Python 2.6 compatability
+	* xdis/op_imports.py: Python 2.6 compatibility
 
 2021-02-27  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/pyenv-newer-versions, xdis/magics.py, xdis/version.py: 
+	* admin-tools/pyenv-newer-versions, xdis/magics.py, xdis/version.py:
 	Add 3.8.8 3.9.2...  Bump version number into dev. Test using newer versions
 
 2021-02-27  rocky <rocky@dusytfeet.com>
@@ -3093,7 +3093,7 @@
 
 2021-01-11  rocky <rocky@dusytfeet.com>
 
-	* xdis/magics.py, xdis/op_imports.py, xdis/opcodes/opcode_37pypy.py: 
+	* xdis/magics.py, xdis/op_imports.py, xdis/opcodes/opcode_37pypy.py:
 	Add pypy3.7
 
 2021-01-11  rocky <rocky@dusytfeet.com>
@@ -3107,7 +3107,7 @@
 
 2021-01-09  rocky <rocky@dusytfeet.com>
 
-	* xdis/magics.py: Add Python veresion 3.8.7
+	* xdis/magics.py: Add Python version 3.8.7
 
 2020-12-16  rocky <rocky@dusytfeet.com>
 
@@ -3161,7 +3161,7 @@
 
 2020-08-30  rocky <rocky@dusytfeet.com>
 
-	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py: 
+	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py:
 	Get ready for release 5.0.4
 
 2020-08-30  rocky <rocky@dusytfeet.com>
@@ -3220,7 +3220,7 @@
 
 2020-07-25  rocky <rocky@dusytfeet.com>
 
-	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py: 
+	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py:
 	Get ready for release 5.0.2
 
 2020-07-22  rocky <rb@dustyfeet.com>
@@ -3270,7 +3270,7 @@
 
 2020-06-28  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/make-dist-newer.sh, admin-tools/make-dist-older.sh: 
+	* admin-tools/make-dist-newer.sh, admin-tools/make-dist-older.sh:
 	Administrivia
 
 2020-06-28  rocky <rocky@dusytfeet.com>
@@ -3283,7 +3283,7 @@
 
 2020-06-28  rocky <rocky@dusytfeet.com>
 
-	* NEWS.md, xdis/__init__.py, xdis/load.py, xdis/version.py: 
+	* NEWS.md, xdis/__init__.py, xdis/load.py, xdis/version.py:
 	check_load(): better python source detection...  check_load() previously we relied on the extension being ".py" to
 	detect if a file was Python source. There are many valid Python
 	programs that don't have a ".py" extension.  With this change we compile the passed file and that determines if
@@ -3412,8 +3412,8 @@
 2020-06-21  rocky <rocky@dusytfeet.com>
 
 	* xdis/__init__.py, xdis/bin/pydisasm.py, xdis/bytecode.py,
-	xdis/instruction.py, xdis/main.py, xdis/std.py, xdis/version.py: 
-	Redo disasembler options handling -- * --show-bytes and --asm options removed. Use --asm-format -F
+	xdis/instruction.py, xdis/main.py, xdis/std.py, xdis/version.py:
+	Redo disassembler options handling -- * --show-bytes and --asm options removed. Use --asm-format -F
 	instead.  * asm_fmt is now a string choice.  * Don't show integer arg value when there is something more useful
 	  to show. Note that -F=bytes will show this separately
 
@@ -3514,7 +3514,7 @@
 
 	* xdis/opcodes/opcode_23.py, xdis/opcodes/opcode_24.py,
 	xdis/opcodes/opcode_25.py, xdis/opcodes/opcode_26.py: Fill out
-	CALL_FUNCITON_xx formating on 2.x
+	CALL_FUNCITON_xx formatting on 2.x
 
 2020-05-28  rocky <rocky@dusytfeet.com>
 
@@ -3634,7 +3634,7 @@
 
 2020-05-12  rocky <rb@dustyfeet.com>
 
-	* xdis/bin/pydisasm.py, xdis/main.py, xdis/opcodes/opcode_36.py: 
+	* xdis/bin/pydisasm.py, xdis/main.py, xdis/opcodes/opcode_36.py:
 	pydisasm disassembles more Python source now
 
 2020-05-09  rocky <rocky@dusytfeet.com>
@@ -3681,14 +3681,14 @@
 	pytest/test_bytecode.py, pytest/test_disasm.py,
 	pytest/test_opcode.py, pytest/test_stack_effect.py, setup.py,
 	test/test_pyenvlib.py, test/test_pythonlib.py,
-	test_unit/test_disasm.py, xdis/__init__.py, xdis/bin/pydisasm.py: 
+	test_unit/test_disasm.py, xdis/__init__.py, xdis/bin/pydisasm.py:
 	xdis.main -> xdis
 
 2020-04-28  rocky <rocky@dusytfeet.com>
 
 	* xdis/__init__.py, xdis/codetype/__init__.py,
 	xdis/codetype/code30.py, xdis/marsh.py, xdis/op_imports.py: Misc
-	fixes and tweaks...  * export get_opcode_module from xdis * Allow float and string for version and convert where needed.  * Error when unmarshaling a code type on the wrong intepreter
+	fixes and tweaks...  * export get_opcode_module from xdis * Allow float and string for version and convert where needed.  * Error when unmarshaling a code type on the wrong interpreter
 
 2020-04-26  rocky <rocky@dusytfeet.com>
 
@@ -3865,7 +3865,7 @@
 	* pytest/test_opcode.py, xdis/cross_dis.py,
 	xdis/opcodes/opcode_2x.py, xdis/opcodes/opcode_36.py,
 	xdis/opcodes/opcode_37.py, xdis/opcodes/opcode_38.py,
-	xdis/opcodes/opcode_39.py, xdis/opcodes/opcode_3x.py, xdis/std.py: 
+	xdis/opcodes/opcode_39.py, xdis/opcodes/opcode_3x.py, xdis/std.py:
 	Finish stack-effect; test against dis.stack_effect
 
 2020-04-22  rocky <rocky@dusytfeet.com>
@@ -3930,7 +3930,7 @@
 
 2020-04-20  rocky <rocky@dusytfeet.com>
 
-	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py: 
+	* NEWS.md, admin-tools/how-to-make-a-release.md, xdis/version.py:
 	Get ready for release 4.4.0
 
 2020-04-19  rocky <rocky@dusytfeet.com>
@@ -3963,7 +3963,7 @@
 
 2020-04-17  rocky <rb@dustyfeet.com>
 
-	* xdis/codetype/__init__.py, xdis/codetype/code13.py, xdis/main.py: 
+	* xdis/codetype/__init__.py, xdis/codetype/code13.py, xdis/main.py:
 	--asm option fixes
 
 2020-04-17  rocky <rb@dustyfeet.com>
@@ -3973,7 +3973,7 @@
 	pytest/testdata/pypy_lambda.right, pytest/testdata/raise-3.0.right,
 	xdis/codetype/.gitignore, xdis/codetype/__init__.py,
 	xdis/codetype/base.py, xdis/codetype/code38.py: report Code object
-	information better...  Also some lint chainges
+	information better...  Also some lint changes
 
 2020-04-17  rocky <rocky@dusytfeet.com>
 
@@ -4017,7 +4017,7 @@
 
 	* NEWS.md, xdis/codetype/code13.py, xdis/codetype/code15.py,
 	xdis/codetype/code20.py, xdis/codetype/code30.py,
-	xdis/codetype/code38.py, xdis/dropbox/decrypt25.py, xdis/marsh.py: 
+	xdis/codetype/code38.py, xdis/dropbox/decrypt25.py, xdis/marsh.py:
 	Check code types for sanity
 
 2020-04-15  R. Bernstein <rocky@users.noreply.github.com>
@@ -4055,7 +4055,7 @@
 	* pytest/test_codetype.py, xdis/__init__.py, xdis/bytecode.py,
 	xdis/codetype/__init__.py, xdis/codetype/code15.py,
 	xdis/codetype/code20.py, xdis/codetype/code30.py,
-	xdis/codetype/code38.py, xdis/unmarshal.py, xdis/wordcode.py: 
+	xdis/codetype/code38.py, xdis/unmarshal.py, xdis/wordcode.py:
 	Convert code to use the Portable code type...  to_portable() is the main interface.  The bulk of the hard stuff I think is done. unmarshal.py
 	load_code_type is a lot cleaner now.  There is still a bit of residual work, but this commit is large
 	enough as it is.
@@ -4137,7 +4137,7 @@
 
 2020-03-25  rocky <rb@dustyfeet.com>
 
-	* admin-tools/setup-master.sh: Bump master default verison
+	* admin-tools/setup-master.sh: Bump master default version
 
 2020-03-24  rocky <rocky@dusytfeet.com>
 
@@ -4225,7 +4225,7 @@
 	xdis/magics.py, xdis/opcodes/base.py, xdis/opcodes/opcode_36.py,
 	xdis/opcodes/opcode_37.py, xdis/opcodes/opcode_38.py,
 	xdis/opcodes/opcode_39.py, xdis/util.py: Big changes; update
-	versions, FUNCTION_CALL_xx and more...  * Python versions: 3.6.10, 3.7.6. 3.8.1 * Add interpolation of FUNCTION_CALL_{KW,EX} argument for 3.7-3.9 * Fix better_repr() for singleton tuples and lists * Add type info in listing contants
+	versions, FUNCTION_CALL_xx and more...  * Python versions: 3.6.10, 3.7.6. 3.8.1 * Add interpolation of FUNCTION_CALL_{KW,EX} argument for 3.7-3.9 * Fix better_repr() for singleton tuples and lists * Add type info in listing constants
 
 2019-12-22  rocky <rocky@dusytfeet.com>
 
@@ -4341,7 +4341,7 @@
 2019-11-17  rocky <rocky@dusytfeet.com>
 
 	* pytest/test_disasm.py, pytest/test_load_file.py,
-	pytest/test_std.py, requirements-dev.txt, xdis/verify.py: 
+	pytest/test_std.py, requirements-dev.txt, xdis/verify.py:
 	Compatibility for testing from Python 3.1 to 3.4
 
 2019-11-17  rocky <rocky@dusytfeet.com>
@@ -4355,7 +4355,7 @@
 
 2019-11-16  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh: 
+	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh:
 	Administriva - bump testing versions
 
 2019-11-16  rocky <rocky@dusytfeet.com>
@@ -4370,7 +4370,7 @@
 2019-11-12  rocky <rocky@dusytfeet.com>
 
 	* pytest/test_opcode.py, xdis/magics.py,
-	xdis/opcodes/opcode_36pypy.py, xdis/unmarshal.py, xdis/verify.py: 
+	xdis/opcodes/opcode_36pypy.py, xdis/unmarshal.py, xdis/verify.py:
 	PyPy 3.6.9 tolerance
 
 2019-11-12  rocky <rocky@dusytfeet.com>
@@ -4567,7 +4567,7 @@
 2019-10-10  rocky <rocky@dusytfeet.com>
 
 	* pytest/test_disasm.py,
-	pytest/testdata/01_hexstring-2.7-for3x.right, xdis/util.py: 
+	pytest/testdata/01_hexstring-2.7-for3x.right, xdis/util.py:
 	Customize bytecode 1.0-1.2 asm output
 
 2019-10-10  rocky <rocky@dusytfeet.com>
@@ -4632,7 +4632,7 @@
 2019-08-20  rocky <rocky@dusytfeet.com>
 
 	* test_unit/test_dis27.py, test_unit/test_dis33.py,
-	test_unit/test_dis34.py: "keyward pair" -> "named" in unit tests Fixes #51
+	test_unit/test_dis34.py: "keyword pair" -> "named" in unit tests Fixes #51
 
 2019-08-19  rocky <rocky@dusytfeet.com>
 
@@ -4676,7 +4676,7 @@
 
 2019-07-21  rocky <rocky@dusytfeet.com>
 
-	* xdis/code.py, xdis/load.py, xdis/magics.py, xdis/unmarshal.py: 
+	* xdis/code.py, xdis/load.py, xdis/magics.py, xdis/unmarshal.py:
 	Adjust for Python 3.8 It add a posonlyargcount position to the code type.  Therefore we
 	need now a new Code38 code type.
 
@@ -4736,7 +4736,7 @@
 
 2019-06-11  rocky <rocky@dusytfeet.com>
 
-	* xdis/opcodes/base.py, xdis/opcodes/opcode_36.py: 
+	* xdis/opcodes/base.py, xdis/opcodes/opcode_36.py:
 	BUILD_TUPLE_UNPACK_WITH_CALL is a vararg
 
 2019-05-27  R. Bernstein <rocky@users.noreply.github.com>
@@ -4866,7 +4866,7 @@
 2019-01-13  rocky <rb@dustyfeet.com>
 
 	* bin/pydisasm.py, xdis/load.py, xdis/magics.py,
-	xdis/op_imports.py, xdis/opcodes/base.py, xdis/opcodes/opcode_38.py: 
+	xdis/op_imports.py, xdis/opcodes/base.py, xdis/opcodes/opcode_38.py:
 	More 3.8 opcode support
 
 2019-01-13  rocky <rb@dustyfeet.com>
@@ -5142,7 +5142,7 @@
 2018-04-15  rocky <rb@dustyfeet.com>
 
 	* xdis/opcodes/opcode_26pypy.py, xdis/opcodes/opcode_27pypy.py,
-	xdis/opcodes/opcode_32pypy.py, xdis/opcodes/opcode_35pypy.py: 
+	xdis/opcodes/opcode_32pypy.py, xdis/opcodes/opcode_35pypy.py:
 	Correct classification of CALL_METHOD
 
 2018-04-13  rocky <rb@dustyfeet.com>
@@ -5251,7 +5251,7 @@
 
 2018-02-04  rocky <rb@dustyfeet.com>
 
-	* xdis/bytecode.py: Don't use depcrecated call
+	* xdis/bytecode.py: Don't use deprecated call
 
 2018-02-03  rocky <rb@dustyfeet.com>
 
@@ -5267,7 +5267,7 @@
 
 2018-02-03  rocky <rb@dustyfeet.com>
 
-	* xdis/magics.py: Python 2.4 compatiablity
+	* xdis/magics.py: Python 2.4 compatibility
 
 2018-02-01  rocky <rb@dustyfeet.com>
 
@@ -5325,7 +5325,7 @@
 
 2018-01-22  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/setup-master.sh, admin-tools/setup-python-2.4.sh: 
+	* admin-tools/setup-master.sh, admin-tools/setup-python-2.4.sh:
 	Administrivia
 
 2018-01-22  rocky <rocky@dusytfeet.com>
@@ -5476,7 +5476,7 @@
 
 2017-12-09  rocky <rb@dustyfeet.com>
 
-	* xdis/main.py: Python 2.4 compatability
+	* xdis/main.py: Python 2.4 compatibility
 
 2017-12-09  rocky <rb@dustyfeet.com>
 
@@ -5507,7 +5507,7 @@
 
 2017-12-02  rocky <rocky@dusytfeet.com>
 
-	* admin-tools/make-dist-newer.sh, admin-tools/make-dist-older.sh: 
+	* admin-tools/make-dist-newer.sh, admin-tools/make-dist-older.sh:
 	Get ready for release 3.6.2
 
 2017-12-02  rocky <rocky@dusytfeet.com>
@@ -5613,7 +5613,7 @@
 	* NEWS, xdis/opcodes/opcode_pypy26.py,
 	xdis/opcodes/opcode_pypy27.py, xdis/opcodes/opcode_pypy32.py,
 	xdis/opcodes/opcode_pypy35.py, xdis/version.py: Get ready for
-	release 3.6.0 Add opcode_pypyXX compatability Update NEWS
+	release 3.6.0 Add opcode_pypyXX compatibility Update NEWS
 
 2017-09-21  rocky <rocky@dusytfeet.com>
 
@@ -5697,7 +5697,7 @@
 2017-09-04  rocky <rocky@dusytfeet.com>
 
 	* pytest/test_bytecode.py, xdis/bytecode.py, xdis/opcodes/base.py,
-	xdis/opcodes/opcode_36.py, xdis/opcodes/opcode_37.py, xdis/util.py: 
+	xdis/opcodes/opcode_36.py, xdis/opcodes/opcode_37.py, xdis/util.py:
 	bytcode decoding a la 3.6. Add EXTENDED_ARG_SHIFT There are some big and important structural changes here.
 	unpack_opargs_bytecode is similar to unpack_opargs_wordcode of 3.6.  It probably fixes a long-standing but little-noticed bug in how
 	Python 2.x disassembles code; it also modularizes and simplifies
@@ -5769,7 +5769,7 @@
 
 2017-08-15  rocky <rocky@dusytfeet.com>
 
-	* xdis/magics.py, xdis/op_imports.py, xdis/opcodes/opcode_37.py: 
+	* xdis/magics.py, xdis/op_imports.py, xdis/opcodes/opcode_37.py:
 	Python 3.7.0-dev support
 
 2017-08-15  rocky <rocky@dusytfeet.com>
@@ -5818,11 +5818,11 @@
 
 2017-08-12  rocky <rocky@dusytfeet.com>
 
-	* ChangeLog, NEWS, xdis/bytecode.py, xdis/main.py, xdis/wordcode.py: 
+	* ChangeLog, NEWS, xdis/bytecode.py, xdis/main.py, xdis/wordcode.py:
 	Add switch to allow showing dup line numbers Python's disassembly doesn't show line numbers other than the first
 	one when there is more entry for a given line. (That's probably a
 	bug that should be reported?) Right now we show the full deal in pydisasm for asm format.  Here it
-	is imporant since we recreate the line number table based on
+	is important since we recreate the line number table based on
 	information given in the instructions.  We could and probably should allow showing all of the line number in
 	the default format as well.  Underneath this commit adds the parameter to control that.
 
@@ -5941,7 +5941,7 @@
 
 	* xdis/bytecode.py, xdis/wordcode.py: Python 3.6 bytecode/wordcode
 	handling...  Correct JREL offset in findlabels for wordcode instructions Simplify
-	findlabels. I think this is right, but not totally sure.  bytecode.py: distiguish names bytecode (string) from code (object)
+	findlabels. I think this is right, but not totally sure.  bytecode.py: distinguish names bytecode (string) from code (object)
 	better
 
 2017-07-10  rocky <rb@dustyfeet.com>
@@ -6028,7 +6028,7 @@
 
 2017-07-06  rocky <rb@dustyfeet.com>
 
-	* xdis/code.py, xdis/marsh.py: Python cross-version compatable
+	* xdis/code.py, xdis/marsh.py: Python cross-version compatible
 	marshalling
 
 2017-07-05  rocky <rb@dustyfeet.com>
@@ -6054,7 +6054,7 @@
 2017-07-03  rocky <rb@dustyfeet.com>
 
 	* test_unit/test_dis34.py, xdis/bytecode.py: in asm mode turn
-	relative jumps into labels Each instruction now has an intruction size field: inst_size
+	relative jumps into labels Each instruction now has an instruction size field: inst_size
 
 2017-07-02  rocky <rb@dustyfeet.com>
 
@@ -6166,7 +6166,7 @@
 	xdis/opcodes/opcode_32.py, xdis/opcodes/opcode_33.py,
 	xdis/opcodes/opcode_34.py, xdis/opcodes/opcode_35.py,
 	xdis/opcodes/opcode_36.py, xdis/opcodes/opcode_3x.py: Start adding
-	formating of EXTENDED_ARG argument
+	formatting of EXTENDED_ARG argument
 
 2017-05-18  rocky <rocky@dusytfeet.com>
 
@@ -6263,9 +6263,9 @@
 
 	* Makefile, pytest/test_opcode.py, test/Makefile,
 	test_unit/Makefile, test_unit/test_opcode.py,
-	xdis/opcodes/opcode_35.py, xdis/opcodes/opcode_3x.py: 
+	xdis/opcodes/opcode_35.py, xdis/opcodes/opcode_3x.py:
 	SETUP_ASYNC_WITH is jrel ...  found by pytest/test_opcode.py opcode_35.py: correct SETUP_ASYNC_WITH Makefile: add clean target
-	opcode_3x.py: small formating change
+	opcode_3x.py: small formatting change
 
 2017-03-15  R. Bernstein <rocky@users.noreply.github.com>
 
@@ -6292,7 +6292,7 @@
 
 	* Makefile: Change `make unittest` to run `setup.py test`.  That will make the way tests are run by robots (e.g. by TravisCI)
 	similar to way they are run by humans.  Also it seems that currently most tests in `test_unit/` and all
-	tests in `pytest/` are not run by continious integration.  This
+	tests in `pytest/` are not run by continuous integration.  This
 	commit fixes that.
 
 2017-03-08  Kirill Spitsyn <ksp@google.com>
@@ -6309,7 +6309,7 @@
 2017-03-08  Kirill Spitsyn <ksp@google.com>
 
 	* pytest/{test_load.py => test_load_file.py}: Rename test file to
-	avoid confusing `py.test`.  The similary named files in `pytest` and `test_unit` directory were
+	avoid confusing `py.test`.  The similarly named files in `pytest` and `test_unit` directory were
 	causing `py.test` to fail with: % py.test ============================ test session starts
 	============================= platform linux2 -- Python 2.7.6,
 	pytest-3.0.6, py-1.4.32, pluggy-0.4.0 rootdir:
@@ -9820,7 +9820,7 @@
 	xdis/opcodes/opcode_23.py, xdis/opcodes/opcode_24.py,
 	xdis/opcodes/opcode_25.py, xdis/opcodes/opcode_26.py,
 	xdis/opcodes/opcode_27.py, xdis/opcodes/opcode_33.py,
-	xdis/opcodes/opcode_pypy26.py, xdis/opcodes/opcode_pypy27.py: 
+	xdis/opcodes/opcode_pypy26.py, xdis/opcodes/opcode_pypy27.py:
 	Another round of DRYing code
 
 2017-01-14  rocky <rocky@dusytfeet.com>
@@ -10009,7 +10009,7 @@
 
 2016-10-27  rocky <rb@dustyfeet.com>
 
-	* xdis/opcodes/opcode_30.py, xdis/opcodes/opcode_31.py: 
+	* xdis/opcodes/opcode_30.py, xdis/opcodes/opcode_31.py:
 	Clarification of EXTENDED_ARG in 3.0 and 3.1
 
 2016-10-25  rocky <rb@dustyfeet.com>
@@ -10256,7 +10256,7 @@
 	* Makefile, test/test_pyenvlib.py,
 	test_unit/2.7/test_dis.py-notyet, test_unit/{test_dis.py-notyet =>
 	test-dis.py}, test_unit/{test_disasm.py-nottyet =>
-	test_disasm.py-notyet}, xdis/dropbox/decrypt25.py, xdis/marsh.py: 
+	test_disasm.py-notyet}, xdis/dropbox/decrypt25.py, xdis/marsh.py:
 	Python 3 compliance and tests Python 3 compliance in marsh.py and decrypt25.py, but more work is
 	needed.  Add more tests, however fragmented
 
@@ -10299,7 +10299,7 @@
 
 2016-08-26  rocky <rocky@dusytfeet.com>
 
-	* test/test_pyenvlib.py, xdis/opcodes/opcode_31.py, xdis/verify.py: 
+	* test/test_pyenvlib.py, xdis/opcodes/opcode_31.py, xdis/verify.py:
 	Python 3.1 opcode bugs ...  verify assert statement.  test_pyenvlib.py: Allow 3.1 testing
 
 2016-08-26  rocky <rocky@dusytfeet.com>
@@ -10432,7 +10432,7 @@
 
 2016-07-25  rocky <rocky@dusytfeet.com>
 
-	* xdis/opcodes/opcode_pypy27.py, xdis/opcodes/opcode_pypy32.py: 
+	* xdis/opcodes/opcode_pypy27.py, xdis/opcodes/opcode_pypy32.py:
 	Document PyPY opts better And fix a bog in JUMP_IF_NOT_DEBUG jrel_op call
 
 2016-07-24  rocky <rocky@dusytfeet.com>
@@ -10502,7 +10502,7 @@
 	xdis/opcodes/opcode_33.py, xdis/opcodes/opcode_34.py,
 	xdis/opcodes/opcode_35.py, xdis/opcodes/opcode_36.py,
 	xdis/opcodes/opcode_3x.py, xdis/opcodes/opcode_pypy26.py,
-	xdis/opcodes/opcode_pypy27.py, xdis/util.py, xdis/wordcode.py: 
+	xdis/opcodes/opcode_pypy27.py, xdis/util.py, xdis/wordcode.py:
 	Moagstar's Python 3.6 support With some small changes - fix some of my typos - handle a merge conflict in xdis/byteocde
 
 2016-07-20  rocky <rocky@dusytfeet.com>
@@ -10606,7 +10606,7 @@
 
 2016-05-31  rocky <rocky@dusytfeet.com>
 
-	* xdis/main.py: Reduce uneccesary imports of unused opcodes
+	* xdis/main.py: Reduce unnecessary imports of unused opcodes
 
 2016-05-31  rocky <rocky@dusytfeet.com>
 
@@ -10662,7 +10662,7 @@
 2016-05-26  rocky <rocky@dusytfeet.com>
 
 	* ChangeLog, NEWS, test/Makefile, test/test_pyenvlib.py: Get ready
-	for relase 1.00 test/test_pyenv.lib: Python 2.5 tolerance.
+	for release 1.00 test/test_pyenv.lib: Python 2.5 tolerance.
 
 2016-05-26  rocky <rocky@dusytfeet.com>
 
@@ -10763,12 +10763,12 @@
 
 	* .gitignore, pyxdis/main.py, pyxdis/opcodes/opcode_26.py,
 	pyxdis/opcodes/opcode_27.py, pyxdis/opcodes/opcode_pypy27.py,
-	test/simple_source/05_24_float.py, test_unit/test_opcode.py: 
+	test/simple_source/05_24_float.py, test_unit/test_opcode.py:
 	Separate PYPY 2.7 from 2.7; DRY 27
 
 2016-05-22  rocky <rocky@dusytfeet.com>
 
-	* README.rst, pyxdis/opcodes/opcode_32.py, test_unit/test_load.py: 
+	* README.rst, pyxdis/opcodes/opcode_32.py, test_unit/test_load.py:
 	Some tolerance for pypy
 
 2016-05-22  rocky <rocky@dusytfeet.com>
@@ -10791,14 +10791,14 @@
 	* pyxdis/__init__.py, pyxdis/load.py, pyxdis/main.py,
 	pyxdis/marsh.py, pyxdis/opcodes/opcode_23.py,
 	pyxdis/opcodes/opcode_24.py, pyxdis/opcodes/opcode_25.py,
-	pyxdis/opcodes/opcode_26.py, pyxdis/util.py, test/test_pyenvlib.py: 
+	pyxdis/opcodes/opcode_26.py, pyxdis/util.py, test/test_pyenvlib.py:
 	Allow bytecodes back to 2.3 or so. and...  Add timestamps when disassembly Rewrite to allow some tolerance to
 	older Pythons
 
 2016-05-22  rocky <rocky@dusytfeet.com>
 
 	* README.rst, pyxdis/load.py, pyxdis/main.py, pyxdis/util.py,
-	test/test_pyenvlib.py: Correct command-line invocation and...  Improve function infomation: * Split out positional argument names and local variables * Remove duplicate <modle> info.  * comment *all* meta fields * add timestamp info
+	test/test_pyenvlib.py: Correct command-line invocation and...  Improve function information: * Split out positional argument names and local variables * Remove duplicate <module> info.  * comment *all* meta fields * add timestamp info
 
 2016-05-22  rocky <rocky@dusytfeet.com>
 
@@ -10874,7 +10874,7 @@
 
 2016-05-19  rocky <rb@dustyfeet.com>
 
-	* .travis.yml, test_unit/test_load.py, test_unit/test_opcode.py: 
+	* .travis.yml, test_unit/test_load.py, test_unit/test_opcode.py:
 	More opcodes, more travis testing.
 
 2016-05-19  rocky <rb@dustyfeet.com>
@@ -10895,4 +10895,3 @@
 2016-05-19  rocky <rocky@dusytfeet.com>
 
 	* Start Python cross disasemble/marshal project For older history see uncompyle6
-
